watertap/examples/flowsheets/case_studies/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -530,58 +531,113 @@ def initialize_system(m):
     for o in order:
         print(o[0].name)
 
-    # Initial guesses for flow into first reactor
-    tear_guesses = {
-        "flow_vol": {0: 1.2368},
-        "conc_mass_comp": {
-            (0, "S_A"): 0.0007,
-            (0, "S_F"): 0.000429,
-            (0, "S_I"): 0.05745,
-            (0, "S_N2"): 0.0534,
-            (0, "S_NH4"): 0.0092,
-            (0, "S_NO3"): 0.00403,
-            (0, "S_O2"): 0.00192,
-            (0, "S_PO4"): 0.0123,
-            (0, "S_K"): 0.373,
-            (0, "S_Mg"): 0.023,
-            (0, "S_IC"): 0.135,
-            (0, "X_AUT"): 0.1382,
-            (0, "X_H"): 3.6356,
-            (0, "X_I"): 3.2611,
-            (0, "X_PAO"): 3.3542,
-            (0, "X_PHA"): 0.089416,
-            (0, "X_PP"): 1.1127,
-            (0, "X_S"): 0.059073,
-        },
-        "temperature": {0: 308.15},
-        "pressure": {0: 101325},
-    }
-
-    tear_guesses2 = {
-        "flow_vol": {0: 0.003},
-        "conc_mass_comp": {
-            (0, "S_A"): 0.10,
-            (0, "S_F"): 0.16,
-            (0, "S_I"): 0.05745,
-            (0, "S_N2"): 0.039,
-            (0, "S_NH4"): 0.034,
-            (0, "S_NO3"): 0.0028,
-            (0, "S_O2"): 0.00136,
-            (0, "S_PO4"): 0.0254,
-            (0, "S_K"): 0.379,
-            (0, "S_Mg"): 0.0267,
-            (0, "S_IC"): 0.078,
-            (0, "X_AUT"): 0.342,
-            (0, "X_H"): 23.4,
-            (0, "X_I"): 11.5,
-            (0, "X_PAO"): 10.3,
-            (0, "X_PHA"): 0.0044,
-            (0, "X_PP"): 2.76,
-            (0, "X_S"): 3.83,
-        },
-        "temperature": {0: 308.15},
-        "pressure": {0: 101325},
-    }
+    if bio_P:
+        # Initial guesses for flow into first reactor
+        tear_guesses = {
```

Comment:
How did you arrive at these new guesses?

Reply:
I got the initial guesses from Chenyu's ElectroN-P flowsheet, and then I made some very slight modifications to them.